data/jakarta/utils.sh

```diff
@@ -126,3 +126,97 @@ snap_remove()
     snap remove edgexfoundry || true
 }
 
+# wait for services to come online
+# NOTE: this may have to be significantly increased on arm64 or low RAM platforms
+# to accomodate time for everything to come online
+snap_wait_all_services_online()
+{
+    regex="4[0-9][0-9]|5[0-9][0-9]"
+    all_services_online=false
+    i=0
+
+    while [ "$all_services_online" = "false" ];
+    do
```

Comment:
it would be good to print out some logs about what is happening here and the current iteration number for when tests are ran with high verbosity.

Reply:
Thanks! Added.